spec/helpers-spec.coffee

```diff
@@ -36,3 +36,16 @@ describe "The Linter Validation Helper", ->
       lint: ->
     }
     expect(Helpers.validateLinter(linter)).toEqual(true)
+
+describe "The Command Execution Helper", ->
+  beforeEach ->
+    waitsForPromise ->
+      @command = Helpers.exec('echo')
+  it "should throw when no parameters are passed.", ->
+    expect( -> Helpers.exec()).toThrow()
+
+describe "The Command and FilePath Helper", ->
+  it "should throw when no parametes are passed.", ->
+    expect( -> Helpers.execFilePath()).toThrow()
+  it "should throw when no File Path is passed.", ->
+    expect( -> Helpers.execFilePath('echo')).toThrow()
```

Comment:
Yeah. It was giving me trouble earlier, but I can give it another go now. I'll probably get it to work this time.